apps/api/src/common/constants/notifications.ts

```diff
@@ -18,6 +18,7 @@ export const ChannelType = {
   SMS_KAPSYSTEM: 8,
   PUSH_SNS: 9,
   VC_TWILIO: 10,
+  SMS_SNS: 11,
```

Comment:
Since this PR was opened after https://github.com/OsmosysSoftware/osmo-x/pull/305, can you use `SMS_SNS: 12`?

The other PR will be updated to use the 11 value. This is needed so that migrations don't go out of sync and cause issues when PRs are merged in order.

Reply:
Done